mcs/class/System.Net.Http/UnitTests/Makefile

```diff
@@ -0,0 +1,17 @@
+thisdir = class/System.Net.Http/UnitTests
```

Comment:
Why this cannot be in System.Net.Http Makefile?

Reply:
The `UnitTests` certainly cannot be in the top-level `Makefile`.  They are using a "mock" implementation and not the "real" one - so they need to explicitly *not* link against the `System.Net.Http` assembly; which means they need to be kept in a separate assembly from the other tests.

We could, in theory, move the `FunctionalTests` tests into the top-level Makefile, though I think it's cleaner to have them both in subdirectories.

Reply:
The tests are always in separate assembly, so it should not matter. Where do you build the mock assembly?

@akoeplinger will this work with helix/sdks packaging?